### ISSUE TITLE
ustvgo work with this mod.

### ISSUE THREAD

**Issue by jigoop1** [Tempest0580/tempest#95]
@Tempest0580 
pl merge this changes to master.
in file tempest.py
line # 471
ustvgo.ustvgo().play(url)
change to 
ustvgo.ustvgo().play(url, query)

and replace following 2 file in indexers folder
[ustvgo.zip](https://github.com/Tempest0580/tempest/files/6444166/ustvgo.zip)

**Comment by bjgood** [comment]
Yes, it indeed does work. Thanks for your efforts. :). I like this one, over the other one posted here, because it has icons for most of the channels.

Have a good one.
Later...

**Comment by bjgood** [comment]
Sorry to report that this update, by jigoop1, has stopped working (at least for me).
